utils/policies-roles-mapping.json

```diff
@@ -514,7 +519,12 @@
         "false": [
             {
                 "roleName": "team_user",
-                "permission": "manage_webhooks",
+                "permission": "manage_incoming_webhooks",
+                "shouldHave": true
+            },
+            {
+                "roleName": "team_user",
+                "permission": "manage_outgoing_webhooks",
```

Comment:
I can't remember if a mapping on the webapp needs to be updated too?

Reply:
The mappins are used in the webapp admin console to map from config flags to permissions settings, and in the backend during the migration, we are okey to migrate it directly from the previous old version, so I think there is no harm in having it this way.